gigalixir/routers/windows.py

```diff
@@ -0,0 +1,11 @@
+from gigalixir.shell import cast
+import logging
+
+class WindowsRouter(object):
+    def route_to_localhost(self, ip, epmd_port, distribution_port):
+        logging.getLogger("gigalixir-cli").info("Setting up route")
+        cast("route ADD %s MASK 255.255.255.255 127.0.0.1" % ip)
```

Comment:
I'm not that familiar with windows, were you able to get observer working with these route commands?

Reply:
I just gave it a shot (I hadn't been using observers) and it doesn't appear to be working. It gets hung up on "Fetching erlang cookie" and I also had a "getsockname failed: Not a socket" message above that, which doesn't sound right.
Can I ask what is the expected output/behavior of the command? I'm not actually sure what I should be seeing if it works properly.

Reply:
Here's the output I get on linux. Which app are you testing this against? Is it using distillery?
```
gigalixir ps:observer
Setting up SSH multiplexing master
Waiting for SSH multiplexing master
Warning: Permanently added '[us-central1.gcp.ssh.gigalixir.com]:32668,[35.192.76.220]:32668' (ECDSA) to the list of known hosts.
Fetching erlang cookie
Using erlang cookie: REDACTED
Fetching pod ip
Using pod ip: 10.1.11.101
Using node name: api
Fetching epmd port and app port.
Setting up SSH tunnel for ports 42192 and 4369
Routing 10.1.11.101 to 127.0.0.1
Setting up iptables
If prompted, please enter your sudo password:
[sudo] password for js: 
Running observer using: erl -name 5dd56988-7358-4c9a-8943-e2c95c44d1d2@10.1.11.101 -setcookie REDACTED -hidden -run observer


============
Instructions
============
In the 'Node' menu, click 'Connect Node'
enter: api@10.1.11.101
and press OK.


Erlang/OTP 21 [erts-10.0] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe]

Eshell V10.0  (abort with ^G)
(5dd56988-7358-4c9a-8943-e2c95c44d1d2@10.1.11.101)1> 
User switch command
 --> q
Cleaning up route from 10.1.11.101 to 127.0.0.1
Cleaning up iptables
If prompted, please enter your sudo password:
Cleaning up SSH multiplexing master
Deleting SSH multiplexing file
```

Reply:
The app I'm testing with is lovely-creepy-westafricanantelope. It is using Distillery, but it is free tier on everything if that makes a difference.
I dug in a little more and the error I'm seeing is coming from the ssh command when I go to start the observer.
`ssh  -S /tmp/gigalixir-cm-fb2fac55-ed24-4c7c-89ab-1bb99f3f44e0 root@us-central1.gcp.ssh.gigalixir.com -p 30722 -N -M`
It actually never gets to the Windows.Observer changes that were made.

I'm not sure what the problem is, but I also don't use ssh normally. As a guess, I tried changing the generated tmp path to a Windows style directory, but that didn't seem to make a difference.

Funnily enough though, when I was testing the other stuff I do use I didn't have any problems, and I know some of those use ssh. I've successfully been able to:
`ps:migrate`
`account:ssh_keys:add`
`ps:remote_console`
`apps`
`pg`
as well as a few others (login, logout, etc)

Reply:
I think maybe the problem is the `-S /tmp/foo` filename. Maybe it needs to be changed to a more windows friendly path such as `%TEMP%\foo`. I'm not familiar with windows, but I found that here https://stackoverflow.com/questions/15111103/tmp-folder-in-windows-like-tmp-in-linux

It makes sense that `ps:remote_console`, etc wouldn't have this problem since they don't use ssh multiplexing i.e. the `-S /tmp/foo` flag.

Reply:
From what I can tell, ControlPath just isn't supported on Windows maybe.
https://github.com/microsoft/vscode-remote-release/issues/629

I can take out the ControlParth argument to get past that bit, but it obviously fails shortly after.

Perhaps the best bet is just to put in a message for Windows users that this specific functionality is not supported on their platform?

Reply:
That sounds good to me. Thanks for investigating! I'll merge as soon as the message is in place.